.github/ISSUE_TEMPLATE/BUG_REPORT.md

```diff
@@ -0,0 +1,53 @@
+<!--
+  --- IMPORTANT ---
+  This is a template for a bug report! If you want to submit a feature request,
+  please paste this link into your browser and follow the instructions there.
+
+  https://github.com/gitpoint/git-point/issues/new?template=FEATURE_REQUEST.md
+  -----------------
+-->
+
+<!--
+  Hi there!
+
+  Thanks for considering to file a bug with GitPoint. Please take a moment to
+  answer the basic questions listed in this template. If there is no need for
+  certain fields or sections, please delete those headers before submitting. We
+  know not all tickets require those steps. Otherwise, please try to be as
+  detailed as possible.
+
+  If this is just a generic question, please consider talking with us on Gitter:
+  https://gitter.im/git-point
+
+  Thanks!
+-->
+
+## Phone OS and Model
+
+<!--
+  Please specify the OS version and model of phones that this bug has been
+  tested on.
+-->
+
+## My Reproduction Steps
+
+<!--
+  Please specify the exact steps you took for this bug to occur. Provide as much
+  detail as possible so we're able to reproduce these steps.
+-->
+
+1. XXX
+1. XXX
+1. XXX
+1. XXX
+
+## Actual vs. Expected
+
+<!--
+  Describe what happened. What did you expect to happen?
+-->
+
+
+<!-- DO NOT MODIFY BELOW THIS LINE -->
+--------------------------------------
+<!-- GITPOINT_BUG -->
```

Comment:
We can use this for auto tagging issues in the future if we make some kind of GitPoint bot.